plugins/modules/win_command.py

```diff
@@ -116,6 +116,11 @@
       - C:\Program Files\My Application\run.exe
       - argument 1
       - -force
+
+- name: Run a batch script with big5 encoding (fixing garbled text issue)
+  ansible.windows.win_command: C:\somescript.bat
+  args:
+    output_encoding_override: big5
```

Comment:
```suggestion
- name: Run an executable that outputs text with big5 encoding
  ansible.windows.win_command: C:\someprog.exe
  args:
    output_encoding_override: big5
```

I would avoid mentioning it's a `.bat` script and explicitly set in the name that it outputs that encoding. Saying `(fixing garbled text issue)` may indicate to people that using `big5` will fix all those issues when in reality it's there for the rare cases when an executable doesn't use the existing console encoding (which we set to utf-8).

Reply:
I think it's common case in non-English systems, especially in Chinese, Japanese.